src/main/routes/index.ts

```diff
@@ -41,6 +42,9 @@ routes.post(urls.SOLE_TRADER_WHAT_IS_YOUR_NAME, nameValidator, soleTraderNameCon
 routes.get(urls.SOLE_TRADER_SECTOR_YOU_WORK_IN, soleTraderSectorYouWorkInController.get);
 routes.post(urls.SOLE_TRADER_SECTOR_YOU_WORK_IN, sectorYouWorkInValidator, soleTraderSectorYouWorkInController.post);
 
+routes.get(urls.SOLE_TRADER_WHICH_SECTOR_OTHER, whichsectorOtherController.get);
+routes.post(urls.SOLE_TRADER_WHICH_SECTOR_OTHER, whichSectorOtherValidator, whichsectorOtherController.post);
```

Comment:
If you plan to use same controller for all three journeys, can you move the controller to controllers/features/common.
if not rename controller to soleTraderWhichSectorOtherController to differentiate between three controllers